compiler/ztests/sql/scope-from-subquery.yaml

```diff
@@ -0,0 +1,17 @@
+script: |
```

Comment:
Nit: Call this something like subqueries-select-star.yaml so it's grouped with the other subqueries-*.yaml tests.